nameko/rpc.py

```diff
@@ -351,6 +367,33 @@ def __call__(self, *args, **kwargs):
         reply = self._call(*args, **kwargs)
         return reply.result()
 
+    @property
```

Comment:
Yes. It seems this would be helpful (https://github.com/onefinestay/nameko/issues/374)

I'll make this enhancement as part of this PR